test/lit/extern-conversions.wast

```diff
@@ -1,20 +1,40 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 
 ;; Check that extern conversion instructions are emitted properly in the binary format.
+;; Also check that the optimizer does not break on this code.
 
 ;; RUN: wasm-opt %s -all --roundtrip -S -o - | filecheck %s
+;; RUN: wasm-opt %s -all -O1 --roundtrip -S -o - | filecheck %s --check-prefix OPTIM
```

Comment:
We can probably just have the single `RUN` line with `-O1`.

Reply:
Coverage would be the same, yeah, since nothing gets optimized here. I thought it might be more clear to explicitly test both cases, but fair enough, I'll remove.